cmd/ingester/main.go

```diff
@@ -25,6 +26,10 @@ func main() {
 			GRPCMiddleware: []grpc.UnaryServerInterceptor{
 				middleware.ServerUserHeaderInterceptor,
 			},
+			GRPCOptions: []grpc.ServerOption{
+				// We have seen 20MB returns from queries - add a bit of headroom
+				grpc.MaxSendMsgSize(64 * 1024 * 1024),
```

Comment:
What was the previous limit? "Unlimited"?

Reply:
Actually it does look like the default is unlimited.  Maybe we only need this on the receiving side...